nixos/modules/programs/flashprog.nix

```diff
@@ -10,14 +10,10 @@ let
 in
 {
   options.programs.flashprog = {
-    enable = lib.mkOption {
-      type = lib.types.bool;
-      default = false;
-      description = ''
-        Installs flashprog and configures udev rules for programmers
-        used by flashprog.
-      '';
-    };
+    enable = lib.mkEnableOption ''
+      Installs flashprog and configures udev rules for programmers
```

Comment:
This will render "Whether to enable Installs flashprog [...]". I suggest to reword a bit. Also, I think mkEnableOption adds trailing dot.

Reply:
Thanks you. I've rephrased it.